src/app/AppHeader/AppHeader.js

```diff
@@ -1,28 +1,27 @@
 import styled from 'styled-components/macro'
 import Pennant from './Pennant'
 
-export default function AppHeader({className}) {
-
+export default function AppHeader({ className }) {
   return (
-    <HeaderStyled className={className}>
-      <PennantStyled>
-        <Pennant/>
-      </PennantStyled>
-    </HeaderStyled>
+    <Header className={className}>
+      <PennantWrapper>
+        <Pennant />
+      </PennantWrapper>
+    </Header>
   )
 }
 
-const HeaderStyled = styled.header`
-  margin: 0 auto;
+const Header = styled.header`
   position: relative;
   height: 50px;
   max-width: 1240px;
```

Comment:
The same with the width. Maybe you should adjust the `max-width` of your `AppWrapper`?